packages/frontend/src/components/VisualizationConfigs/common/hooks/useControlledAccordion.tsx

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useState } from 'react';
+
+export const useControlledAccordion = (defaultOpenItems = []) => {
```

Comment:
This is an interesting approach. To have a hook instead of a wrapper or something. But I think I like it.